data_diff/dbt.py

```diff
@@ -315,24 +320,24 @@ def get_models(self):
                 f"Found dbt: v{dbt_version} Expected the dbt project's version to be >= {LOWER_DBT_V} and < {UPPER_DBT_V}"
             )
 
-        with open(self.project_dir / MANIFEST_PATH) as manifest:
-            manifest_dict = json.load(manifest)
-            manifest_obj = self.parse_manifest(manifest=manifest_dict)
-
         success_models = [x.unique_id for x in run_results_obj.results if x.status.name == "success"]
-        models = [manifest_obj.nodes.get(x) for x in success_models]
+        models = [self.manifest_obj.nodes.get(x) for x in success_models]
         if not models:
             raise ValueError("Expected > 0 successful models runs from the last dbt command.")
 
-        rich.print(f"Found {str(len(models))} successful model runs from the last dbt command.")
+        rich.print(f"Found {str(len(models))} successful model runs from the last dbt command. Starting diffs.\n")
```

Comment:
words?

Reply:
I agree, it feels awkward as a user to be left hanging when diffs aren't instant.

Looking for additional options, dbt does "Running with dbt=1.4.0" after a run command, maybe "Running with data-diff={version}"

Reply:
That's a good idea @kylemcnair.

It'll also reinforce what version they are on so when we upgrade and talk about it, they'll have a better idea of whether they can access the newest features.

Reply:
I like it

Reply:
updated